src/index.ts

```diff
@@ -649,7 +673,10 @@ export class ValidationService {
 
         for (let i = 0; i < formInputUIDs.length; i++) {
             let inputUID = formInputUIDs[i];
-            formValidators.push(this.validators[inputUID]);
+            const validator = this.validators[inputUID];
+            if (validator) {
+                formValidators.push(validator);
+            }
```

Comment:
Agree it's curious that `inputUID` doesn't seem to have been deleted, but guarding against `undefined` also doesn't both me.